### ISSUE TITLE
DSSIM documentation

### ISSUE THREAD

**Issue by isaacgerg** [keras-team/keras-contrib#477]
What is the input domain of the data for this loss?  [0,1], [-1,1], [0, 255]?

**Comment by iperov** [comment]
it has wrong implementation

try my port of tensorflow https://github.com/keras-team/keras-contrib/issues/464

**Comment by isaacgerg** [comment]
@iperov Thank you.  My question remains the same for your implementation though.  Can you comment?

**Comment by iperov** [comment]
input [0, max_value]

**Comment by isaacgerg** [comment]
@iperov  Thank you.

**Comment by isaacgerg** [comment]
@iperov Just be absolutely clear, your function can take in single channel inputs correct? (luminance only?)  It appears to from going through it.  However, I cannot get keras to resolve the shapes after the reducer step.

**Comment by isaacgerg** [comment]
I am going to reopen this.  @iperov what is wrong with the code in keras_contrib?

**Comment by iperov** [comment]
keras_contrib not implements paper  `Wang, Z., Bovik, A. C., Sheikh, H. R., & Simoncelli, E. P. (2004). Image
  quality assessment: from error visibility to structural similarity. IEEE
  transactions on image processing.`

tensorflow implementation:
https://github.com/tensorflow/tensorflow/blob/f3954bf90088f0a464825307a32b23a84e15dfff/tensorflow/python/ops/image_ops_impl.py#L2940

**Comment by iperov** [comment]
> However, I cannot get keras to resolve the shapes after the reducer step.

brofl

**Comment by iperov** [comment]
good luck then
unsubscribing this shit

**Comment by isaacgerg** [comment]
@iperov  I don't get it.

**Comment by isaacgerg** [comment]
I solve the issue by using tensorflows implementation of multi-scale ssim.  I checked it against the original reference and it appears to be correct.
